static-site/src/components/splash/index.jsx

```diff
@@ -28,13 +28,33 @@ const Section = ({id, title, abstract, cards, buttonText, buttonLink}) => (
         />
       </div>
       <BigSpacer/>
-      <Styles.Button to={buttonLink}>
+      <Button to={buttonLink}>
         {buttonText}
-      </Styles.Button>
+      </Button>
     </div>
   </div>
 );
 
+const Button = ({to, children}) => {
```

Comment:
would it be better to pull this out into a dedicated component file?

Reply:
Dropped this change from the PR